### ISSUE TITLE
Error during installation: "node.js" on Mac OS with nvm

### ISSUE THREAD

**Issue by iamskok** [pichillilorenzo/JavaScriptEnhancements#65]
Hi there 👋 

I really enjoyed using your plugin, but I started getting errors after I switched to `nvm`:

```
Error during installation: "node.js" seems not installed on your system. Node.js (6 or upper) and npm are required to be able to use JavaScript Enhancements plugin.
```

In order to fix it I added new `PATH` in my `.../User/JavaScript Enhancement.sublime-settings` file, but it didn't help.

```
{
	"PATH": ":/Users/skok/.nvm/versions/node/v10.9.0/bin/node"
}
```

JavaScript Enhancement v0.16.20
Sublime Text v3.1.1

**Comment by nate-vukovich** [comment]
Have you tried setting the version?

`nvm use 10.9.0` ?

And setting a default alias will help you in the long run: `nvm alias default 10.9.0` - to se the alias default to use this version - `nvm use default` ...

And make sure after you install the version you open a new terminal window - just to make sure that the path was loaded ...

Let me know if it works out.

**Comment by iamskok** [comment]
@nvwebd  yes, I have already set a default alias of `v10.9.0`, but still get the same error.
Here is the output from `nvm ls`:

```
->      v10.9.0
default -> 10.9.0 (-> v10.9.0)
node -> stable (-> v10.9.0) (default)
stable -> 10.9 (-> v10.9.0) (default)
iojs -> N/A (default)
lts/* -> lts/carbon (-> N/A)
lts/argon -> v4.9.1 (-> N/A)
lts/boron -> v6.14.4 (-> N/A)
lts/carbon -> v8.11.4 (-> N/A)
```

**Comment by iamskok** [comment]
@nvmebd any other ideas?

**Comment by smdabdoub** [comment]
@iamskok Your PATH setting has 'node' included at the end. Remove that and it should be fine:

    {
	"PATH": ":/Users/skok/.nvm/versions/node/v10.9.0/bin"
    }

**Comment by iamskok** [comment]
> @iamskok Your PATH setting has 'node' included at the end. Remove that and it should be fine:
> 
> ```
> {
> "PATH": ":/Users/skok/.nvm/versions/node/v10.9.0/bin"
> }
> ```

Thanks a lot! That was the issue.

**Comment by kapilkaushal24** [comment]
type node -v and got this error

dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/mac/.nvm/versions/node/v18.12.1/bin/node (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/mac/.nvm/versions/node/v18.12.1/bin/node (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

**Comment by kapilkaushal24** [comment]
all the giving solution i tried but o one will work please help 
.
